string-hash/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for string-hash 1.1.1
+// Project: https://github.com/darkskyapp/string-hash
+// Definitions by: Ethan Rubio <https://github.com/ethanrubio/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace StringHash {
```

Comment:
I'm getting a tsc error when I did what you recommended:

`'declare' modifier required for top level element`

Is the following acceptable instead?
```js
declare var stringHash: (str: string) => number;
export = stringHash;
```